src/sentry/monitors/serializers.py

```diff
@@ -115,16 +131,19 @@ def _expand(self, key) -> bool:
         return key in self.expand
 
 
-class MonitorSerializerResponse(TypedDict):
+class MonitorCheckInSerializerResponseOptional(TypedDict, total=False):
+    groups: List[str]
```

Comment:
This was originally `group_id`, but it was changed to `groups` in the serializer response without being changed in the TypedDict.